src/pages/recipes/containers/RecipeDetailsContainer.tsx

```diff
@@ -32,6 +34,9 @@ class RecipeDetailsContainer extends React.Component<Props> {
           navigation.pop(1);
         } }
         recipeDetails={ recipeDetails }
+        likeRecipe = { likeRecipe }
+        unlikeRecipe = { unlikeRecipe }
+        favorited = {// how to get this val}
```

Comment:
should a field in the Recipe when fetch, isn't it? you did added it to the data model on backend

Reply:
i don't think so

Reply:
then how did the favorite work on the backend, what kind of flags are added?